src/main/java/core/basesyntax/strategy/impl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private final Map<Operation, OperationHandler> operationHandlers;
+
+    public OperationStrategyImpl(Map<Operation, OperationHandler> operationHandlers) {
+        this.operationHandlers = operationHandlers;
+    }
+
+    public void applyOperation(Operation operation, String fruit,

```

Comment:
The applyOperation method is public but it's not overriding any method from the implemented interface. Please check if this method should be part of the OperationStrategy interface.